app/src/main/res/layout/activity_main.xml

```diff
@@ -31,17 +29,27 @@
             android:layout_height="wrap_content"
             android:text="@string/temp_update_appointment_button"
             style="?android:attr/buttonBarButtonStyle" />
-
     </LinearLayout>
 
+    <Button
+        android:id="@+id/buttonList"
```

Comment:
This is one for the future but we will create style components for the buttons so it is just plug and play. No changes needed here right now

Reply:
As seen in [7a91bc1](https://github.com/priyank1968/CloverEmployeeAppointment/pull/19/commits/7a91bc105cdc8e5076febd0d63dec3df4ab56dd6), removed the button to list appointments. Though I am not sure how to go back to the list appointments fragment.